app/models/assigned_server_role.rb

```diff
@@ -13,6 +13,7 @@ class AssignedServerRole < ApplicationRecord
 
   def massage_active_field
     self.active = false if active.nil?
+    nil
```

Comment:
I'm 99.7% sure this did *not* intend to abort the save when `active.nil?` was true